pitch.md

```diff
@@ -1,8 +1,17 @@
-# Project Title (replace me)
+# Construct
 
 ## Overview
 _What's this project about, briefly?_
 
+The Construct language is a domain specific language for doing geometric
+constructions on the fly.
```

Comment:
That would be pretty neat indeed. It's not currently on my radar, but I'll keep in in the back of my mind.